package.json

```diff
@@ -98,7 +74,8 @@
     "@stablelib/sha256": "^1.0.0",
     "@stablelib/x25519": "^1.0.0",
     "@stablelib/xchacha20poly1305": "^1.0.0",
-    "did-resolver": "^3.0.1",
+    "did-jwt": "^5.0.1",
```

Comment:
Why does it have itself as a dep? 😅

Reply:
good catch!
this is still in progress. fixing it now..